specs/DesignDiscussions/InputLabels.md

```diff
@@ -0,0 +1,279 @@
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.Submit",
+			"title": "OK"
+		}
+	]
+}
+``` 
+#### Non-adjacent labels
+Like HTML “label for”, this approach doesn’t require the label to be near the input it labels. In most cases, it seems likely that this is not something we want or need to support, and may in fact be confusing to users. However, we should consider if there are valid scenarios for this use case. Possible scenarios that I could come up with include:
+
+##### Labels to the left of the input
+
+![img](assets/InputLabels/LabelsToTheLeft.png)
+
+I built this card using a series of column sets. Although it is an acceptable UI, this is probably not the approach we want for card authors to label their inputs on the side. The card authoring is fairly non-intuitive to get a good result. If we want to support labels to the left of the input, we should probably do this as a first-class explicit property (```"labelPosition"``` or ```"allowHorizontalLabels"```). 
+
+##### Additional content between the label and the input
+![img](assets/InputLabels/AdditionalLabelContent.png)
+
+This scenario could be covered by a RichTextBlock that contains both pieces of text being used as the label (arguably the entire text should be read as the accessibility label for the input). Are there scenarios where we would want something else (an image?) between the label and the input?
+
+#### Usability/Adoption
+Since the benefits of associating the label and text are somewhat subtle, adoption may be an issue with Option 2. Option 1 has the benefit of being concise and simple for the card author and therefore more likely to be used. If an author wants a simple text label on their input, they can use the label property more easily than generating a separate text box. This would make it more likely that they do so, and therefore get the benefits of accessibility, better interaction model, and possible future responsiveness features even if they themselves don’t consider those issues.
+
+## Placeholders
+Our primary input scenario card uses placeholders to label the inputs. With either of the approached above, how do we support scenarios where the desired behavior is for the inputs to be labeled by their placeholder text?
+
+``` json
+{
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+        {
+            "type": "Input.Text",
+            "placeholder": "Name",
+            "style": "text",
+            "id": "SimpleVal"
+        },
+        {
+            "type": "Input.Text",
+            "placeholder": "Homepage",
+            "style": "Url",
+            "id": "UrlVal"
+        },
+        {
+            "type": "Input.Text",
+            "placeholder": "Email",
+            "style": "Email",
+            "id": "EmailVal"
+        }
+    ],
+    "actions": [
+        {
+            "type": "Action.Submit",
+            "title": "Submit"
+        }
+    ]
+}
+```
+
+![img](assets/InputLabels/Placeholders.png)
```

Comment:
Alberto: Formatted placeholders likely wouldn't work

Reply:
Added comments to the spec about this case.

---
In reply to: [347043525](https://github.com/microsoft/AdaptiveCards/pull/3618#discussion_r347043525) [](ancestors = 347043525)